code/modules/multiz/movement.dm

```diff
@@ -66,69 +75,45 @@
 /mob/living/carbon/human/can_overcome_gravity()
```

Comment:
add in a check for if the mob is using wallrun here

Reply:
> add in a check for if the mob is using wallrun here

ready